tsconfig.json

```diff
@@ -14,7 +14,8 @@
         "composite": true,
         "declaration": true,
         "declarationMap": true,
-        "sourceMap": true
+        "sourceMap": true,
+        "allowJs": true
```

Comment:
@albrow This setting allows us to include Javascript files along with our Typescript. `tsc` generates the appropriate declaration and map files.

This allowed me to avoid converting `wasm_exec.js` to a Typescript file. Any reason why you think this is a bad idea?

Reply:
It has been moved to `@0x/mesh-browser-lite`'s `tsconfig.json` now, but this comment is still relevant.